vmu.py

```diff
@@ -33,48 +33,32 @@ def load_run_params(param_dir):
             # skip non-yaml files
             pass
         else:
-            if param_contents.has_key('platform') \
-               and param_contents.has_key('sources') \
-               and param_contents.has_key('packages'):
+            if set(param_contents.keys()) == set(['platforms', 'sources', 'package_sets']):
```

Comment:
So, it's really just about `dict` in this case, it's not specific to `set`.

So for instance: `for key in param_contents:`
is identical to: `for key in param_contents.iterkeys():`

If you actually want keys and values, you have to do something like:
`for key,val in param_contents.items():`  (or `.iteritems()`)

(Not that it's wrong to do `.keys()` but anyway I bring it up because it seems worth knowing how dict iteration works...)